config/locales/en.yml

```diff
@@ -134,6 +134,7 @@ en:
     name: 'name'
     role: 'role'
     company: 'company'
+    username: "%{user_name}"
```

Comment:
No need to put it in internationalization if it does not contain any text. This is just outputting the value of the variable.